OpenRA.Mods.Common/Traits/Harvester.cs

```diff
@@ -183,8 +183,8 @@ public Actor ClosestProc(Actor self, Actor ignore)
 
 					var occupancy = refs[loc].Occupancy;
 
-					// 4 harvesters clogs up the refinery's delivery location:
-					if (occupancy >= 3)
+					// 3 harvesters clogs up the refinery's delivery location:
+					if (occupancy >= 2)
```

Comment:
@chrisforbes thats the problem what this PR tries to solve, normaly harvesters should search the next free Refinery not the nearest refinery, (regardless if blocked or not)